applications/luci-app-v2raya/Makefile

```diff
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2024 ImmortalWrt.org
+
+include $(TOPDIR)/rules.mk
+
+LUCI_TITLE:=LuCI support for v2rayA
+LUCI_DEPENDS:=+v2raya
+LUCI_PKGARCH:=all
+
```

Comment:
As the submitter of this PR, do you assume responsibility for maintenance and bug fixes of this app?

`PKG_MAINTAINER:=Tianling Shen <cnsztl@immortalwrt.org>`

Reply:
sure